scripts/functions

```diff
@@ -1183,6 +1183,14 @@ CT_DoConfigSub() {
     fi
 }
 
+# Normally, each step is executed in a sub-shell and thus cannot modify the
+# environment for the next step(s). When this is needed, it can do so by
+# invoking this function.
+# Usage: CT_EnvModify VAR VALUE
+CT_EnvModify() {
+    echo "${1}=\"${2}\"" >> "${CT_BUILD_DIR}/env.modify.sh"
+}
```

Comment:
After discussing on IRC, the current proposal is better than mine.

Modifying the environment in that way already allows passing options to following steps, but even more, the so-modified environment is automatically saved for the save/restore mechanism.